src/test/details.test.js

```diff
@@ -0,0 +1,27 @@
+// eslint-disable-next-line import/no-unresolved
```

Comment:
- [ ] Linters help us write better code. Let's maintain that. Kindly remove eslint disabling here and correct the resulting error(s).